diff --git a/cocoa/RNXBridgelessHeaders.h b/cocoa/RNXBridgelessHeaders.h
index f9481e324d58c20b01924b9d6da85e38cbb41e16..1b10a050283d4b165b304259a14db4eeb608e109 100644
--- a/cocoa/RNXBridgelessHeaders.h
+++ b/cocoa/RNXBridgelessHeaders.h	
@@ -21,6 +21,7 @@
 #endif  // USE_REACT_NATIVE_CONFIG
 
 #ifdef USE_FEATURE_FLAGS
+#import <react/featureflags/ReactNativeFeatureFlags.h>
 #ifdef USE_FEATURE_FLAGS_OSS_OVERRIDES
 #include <react/featureflags/ReactNativeFeatureFlagsOverridesOSSStable.h>
 #else  //  USE_FEATURE_FLAGS_OSS_OVERRIDES
